doc/faqs.md

```diff
@@ -1,5 +1,14 @@
 # FAQs
 
+## Debug
+
+### Q: Experiment not working after apply chaos
+You can following the steps to debug:
+- First using `kubectl describe` the chaos experiment resource. If see `NextStart`  and `NextRecover`  field in spec, then after `NextStart` the chaos will trigger.
+- Then use `kubectl logs -f -n chaos-testing chaos-controller-manager-xxxxx(replace this will controller-manager's pod name)`  to get controller-manager's log and see whether there are errors in it, If there are some related errors, please file an issue.
```

Comment:
`kubectl logs -n chaos-testing chaos-controller-manager-xxxxx(replace this will controller-manager's pod name) | grep "ERROR"` ?

Reply:
greet